pkg/registry/apis/provisioning/resources/repository.go

```diff
@@ -23,7 +24,7 @@ type RepositoryResources interface {
 	EnsureFolderExists(ctx context.Context, folder Folder, parentID string) error
 	EnsureFolderTreeExists(ctx context.Context, ref, path string, tree FolderTree, fn func(folder Folder, created bool, err error) error) error
 	// File from Resource
-	CreateResourceFileFromObject(ctx context.Context, obj *unstructured.Unstructured, options WriteOptions) (string, error)
+	WriteResourceFileFromObject(ctx context.Context, obj *unstructured.Unstructured, options WriteOptions) (string, error)
```

Comment:
renaming because this uses the "Write" command (not Create)

Write is good -- in migrate, items from history are written on-top of previous values